demos/projects/ESPRESSIF/az-ca-recovery/README.md

```diff
@@ -164,7 +166,11 @@ Select `Save` and make note of the Group Enrollment Key.
 
 ## Create a Derived Shared Access Key for Group Enrollment
 
-To use the custom allocation policy with the recovery Azure function, you have to create a derived SAS key from the group enrollment key. To create one, [follow the directions here](https://learn.microsoft.com/azure/iot-dps/concepts-symmetric-key-attestation?tabs=azure-cli#group-enrollments) and save the derived key to be used later. You may use a registration id of your choice.
+To use the custom allocation policy with the recovery Azure function, you have to create a derived SAS key from the group enrollment key. To create one, use the following Azure CLI command or [follow the directions here for other options](https://learn.microsoft.com/azure/iot-dps/concepts-symmetric-key-attestation?tabs=azure-cli#group-enrollments). You may use a registration id of your choice. Make sure to save this for the [RECOVERY] Derived Shared Access Key later. The `--key` parameter should be the group enrollment "Primary Key".
```

Comment:
Nit - recommend removing the `or [follow...]'.  I don't think there's much ambiguity in this path since it's the same cmd line, so we want to take this decision point away from users.  We could have something like at end of this "You can learn more about what this command is doing from [here](learn.ms.com/blah/blah)".